src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -76,7 +76,12 @@ export const CollapsibleNavTop = ({
       <EuiFlexGroup responsive={false} justifyContent="spaceBetween">
         {!shouldShrinkNavigation ? (
           <EuiFlexItem grow={false}>
-            <EuiButtonEmpty flush="both" {...homeLinkProps} onClick={onIconClick}>
+            <EuiButtonEmpty
+              flush="both"
+              {...homeLinkProps}
+              onClick={onIconClick}
+              className="homeIcon"
```

Comment:
```suggestion
              className="navGroupEnabledHomeIcon"
```
Will the class name conflict with the existing homeIcon?

Reply:
updated to `navGroupEnabledHomeIcon` as suggested.